db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="performers")
+    genres = models.ManyToManyField(Genre, related_name="category")
```

Comment:
incorrect related names - check ur checklist
are u sure about `actor.performers.all()` should return all movies with actor ?